### PR TITLE
Add vulnerability form

### PR DESCRIPTION
**Pull request by emilevauge**
### What does this PR do?

This PR adds a link to report vulnerabilities in the documentation.
### Motivation

Implement a responsible disclosure policy.
### More

- [x] Added/updated documentation

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -203,3 +203,8 @@ Using the tiny Docker image:
 ```shell
 docker run -d -p 8080:8080 -p 80:80 -v $PWD/traefik.toml:/etc/traefik/traefik.toml traefik
 ```
+ 
+## Security
+
+We want to keep Traefik safe for everyone.
```
Comment (nmengin) [inline docs/index.md]: WDYT to replace `Traefik` par `Træfik`?

File: docs/index.md [hunk 2]
```diff
@@ -203,3 +203,8 @@ Using the tiny Docker image:
...
+If you've discovered a security vulnerability in Traefik, we appreciate your help in disclosing it to us in a responsible manner, using [this form](https://security.traefik.io).
```
Comment (nmengin) [inline docs/index.md]: WDYT to replace `Traefik` par `Træfik`?

File: docs/index.md (hunk 1 above)
Comment (emilevauge) [reply docs/index.md]: oh sure! Done!

File: docs/index.md (hunk 2 above)
Comment (emilevauge) [reply docs/index.md]: oh sure! Done!

**Pull request merged**